composer/core/algorithm.py

```diff
@@ -92,14 +89,15 @@ def match(self, event: Event, state: State) -> bool:
         Args:
             event (Event): The current event.
             state (State): The current state.
+
         Returns:
             bool: True if this algorithm should run now.
         """
         raise NotImplementedError(f'implement match() required for {self.__class__.__name__}')
 
     @abstractmethod
     def apply(self, event: Event, state: State, logger: Logger) -> Optional[int]:
-        """Applies the algorithm to make an in-place change to the :class:`~.state.State`.
+        """Apply the algorithm to make an in-place change to the :class:`~.state.State`.
```

Comment:
I'm seeing a lot of `Applies` -> `Apply`  style changes here. is it necessary to change all of these? I don't think either is necessarily wrong.

pytorch uses the more narrative `Applies` style
https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/batchnorm.py#L231

numpy and pandas are more declarative.

Reply:
Hmm, good catch -- the google style guide prefers descriptive over imperative, so will adjust the rule for that. Thanks!

https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

Reply:
OK, it's going to be too much of a pain to convert all of them -- I did a find and replace for "apply" and "determine", but let's incrementally convert the rest as we refactor the codebase. I disabled the "imperative" setting in the config, so it won't warn on that anymore.